src/preprocess.py

```diff
@@ -17,36 +17,34 @@
 from feature_spleeter import feature_spleeter
 from feature_tempocnn import feature_tempocnn
 
-config_file = Namespace(**yaml.load(open('config_file.yaml'), Loader=yaml.SafeLoader))
-
 DEBUG = False
 
 
 def compute_audio_repr(audio_file, audio_repr_file, force=False):
     if not force:
         if os.path.exists(audio_repr_file):
-            print('{} exists. skipping!'.format(audio_file))
+            print('{} exists. skipping!'.format(audio_repr_file))
             return 0
 
-    if config['type'] == 'waveform':
-        audio, sr = librosa.load(audio_file, sr=config['resample_sr'])
+    if config['config_train']['feature_type'] == 'waveform':
+        audio = es.MonoLoader(
+            filename=audio_file, sampleRate=config['config_train']['feature_params']['resample_sr'])()
         audio_repr = audio
         audio_repr = np.expand_dims(audio_repr, axis=1)
-
-    elif config['feature_name'] == 'melspectrogram':
+    elif config['config_train']['feature_type'] == 'musicnn-melspectrogram':
         audio_repr = feature_melspectrogram_essentia(audio_file)
-    elif config['feature_name'] == 'vggish':
+    elif config['config_train']['feature_type'] == 'vggish-melspectrogram':
         audio_repr = feature_melspectrogram_vggish(audio_file)
-    elif config['feature_name'] == 'ol3':
+    elif config['config_train']['feature_type'] == 'ol3':
         audio_repr = feature_ol3(audio_file)
-    elif config['feature_name'] == 'spleeter':
+    elif config['config_train']['feature_type'] == 'spleeter':
         audio_repr = feature_spleeter(audio_file)
-    elif config['feature_name'] == 'tempocnn':
+    elif config['config_train']['feature_type'] == 'tempocnn':
         audio_repr = feature_tempocnn(audio_file)
-    elif config['feature_name'] == 'effnet_b0':
+    elif config['config_train']['feature_type'] == 'effnet_b0':
         audio_repr = feature_effnet_b0(audio_file)
     else:
-        raise Exception('Feature {} not implemented.'.format(config['type']))
+        raise Exception('Feature {} not implemented.'.format(config['config_train']['feature_type']))
```

Comment:
I think this could be a concrete Exception like [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError)